pinwheel-android/src/main/java/com/underdog_tech/pinwheel_android/Pinwheel.kt

```diff
@@ -10,21 +10,25 @@ object Pinwheel {
     const val CDN_URL= "https://cdn.getpinwheel.com/link-v2.html"
 
     fun init(webView: WebView, linkToken: String, callback: PinwheelEventListener?) {
+        val timestamp = getUnixTimestamp()
         val uniqueUserId = UUIDManager(webView.context).uuid
-
         uniqueUserId?.let {
-            configureWebView(webView, linkToken, it, callback)
+            configureWebView(webView, linkToken, it, timestamp, callback)
         }
     }
 
     @SuppressLint("SetJavaScriptEnabled")
-    private fun configureWebView(webView: WebView, linkToken: String, uuid: String, callback: PinwheelEventListener?) {
+    private fun configureWebView(webView: WebView, linkToken: String, uuid: String, timestamp: Long, callback: PinwheelEventListener?) {
         webView.settings.javaScriptEnabled = true
-        webView.webViewClient = PinwheelWebViewClient(linkToken, uuid)
+        webView.webViewClient = PinwheelWebViewClient(linkToken, uuid, timestamp)
 
         val pinwheelJSInterface = PinwheelJavaScriptInterface(callback)
         pinwheelJSInterface.bind((webView))
 
         webView.loadUrl(CDN_URL)
     }
+
+    private fun getUnixTimestamp(): Long {
+        return System.currentTimeMillis() / 1000L
+    }
```

Comment:
Is the timestamp returned from `getUnixTimestamp` in milliseconds?

Reply:
Currently in seconds. Will fix in a minute.